dvc/scm/git.py

```diff
@@ -230,3 +238,114 @@ def belongs_to_scm(self, path):
         basename = os.path.basename(path)
         path_parts = os.path.normpath(path).split(os.path.sep)
         return basename == self.ignore_file or Git.GIT_DIR in path_parts
+
+    def get_tree(self, rev):
+        return GitTree(self.git, rev)
+
+
+# see git-fast-import(1)
```

Comment:
Let's move everything GitTree related into a separate file. E.g. `dvc/scm/git/tree.py`.

Reply:
Do you mean, we should split dvc/scm/git.py into `dvc/scm/git/__init__.py` and `dvc/scm/git/tree.py`?

Reply:
@ei-grad Yep :)